docs/api_docs/example-config.json

```diff
@@ -0,0 +1,28 @@
+{
+    "hideMemberFields": [
+        "TypeMeta"
+    ],
+    "hideTypePatterns": [
+        "ParseError$",
+        "List$"
+    ],
```

Comment:
@PrasadG193 should we change name of this file to just `config.json`?

Reply:
@viveksinghggits I changed the file to just `config.json`